src/app/components/ResultsList/ResultsList.jsx

```diff
@@ -89,18 +95,31 @@ const ResultsList = ({
     const hasRequestTable = items.length > 0;
     const { baseUrl } = appConfig;
     const bibUrl = `${baseUrl}/bib/${bibId}`;
+    const { totalPhysicalItems, eResources } = getElectronicResources(result);
+    const resourcesOnClick = () => {
+      updateResultSelection({
+        fromUrl: `${pathname}${search}`,
+        bibId,
+      })
+    }
 
+    const hasPhysicalItems = totalPhysicalItems > 0;
+    const itemCount = hasPhysicalItems ? totalPhysicalItems : eResources.length;
+    const resourceType = hasPhysicalItems ? 'item' : 'resource';
+    const itemMessage = `${itemCount} ${resourceType}${itemCount !== 1 ? 's' : ''}`;
```

Comment:
These four lines would be a good candidate for a new utility function.

Reply:
Good point, but in the interest of keeping this moving I'll leave that to another refactor.